torch/package/package_exporter.py

```diff
@@ -235,7 +235,16 @@ def _get_dependencies(
         package_name = (
             module_name if is_package else module_name.rsplit(".", maxsplit=1)[0]
         )
-        dep_pairs = find_files_source_depends_on(src, package_name)
+        try:
+            dep_pairs = find_files_source_depends_on(src, package_name)
+        except Exception as e:
+            self.dependency_graph.add_node(
+                module_name,
+                error=PackagingErrorReason.DEPENDENCY_RESOLUTION_FAILED,
+                error_context=str(e),
```

Comment:
Since this will eventually become a `PackagingError`, would it be useful to [chain `e` to the eventual `PackagingError` that will be thrown](https://docs.python.org/3/tutorial/errors.html#exception-chaining)?

Since the `PackageError` isn't raised right away, we'd have to store `e` to use in a `raise...from` later. I don't know if that's advised. :S

Reply:
The primary problem is that PackagingError may have a huge number of causes, so it's hard to raise from just one.

Reply:
That's true sigh